portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,33 +33,55 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     public List<String> messages;
+    public List<String> oldComments;
```

Comment:
It is hard to tell what's the difference between messages and oldComments.

In most cases, local variables are preferred over instance variable.  i.e. a smaller scope helps readability.